### PR TITLE
BAU - Update secret values

### PR DESCRIPTION
**Pull request by aaronwilliamsv1**
### Change description

* Updated RSA keys to use a shared key than a specific key

+ref: BAU
+semver: minor

- [ X ] Unit tests and other appropriate tests added or updated
- [ X ] README and other documentation has been updated / added (if needed)
- [ X ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### CODE REVIEW

**Review by gidsg — CHANGES_REQUESTED** [review]
Frontend should not need private key, please remove this.

File: copilot/fsd-frontend/manifest.yml [hunk 1]
```diff
@@ -62,8 +62,8 @@ variables:
     from_cfn: ${COPILOT_APPLICATION_NAME}-${COPILOT_ENVIRONMENT_NAME}-RedisInstanceURI
 
 secrets:
-  RSA256_PUBLIC_KEY_BASE64: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/frontend_RSA256_PUBLIC_KEY_BASE64
-  RSA256_PRIVATE_KEY_BASE64: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/frontend_RSA256_PRIVATE_KEY_BASE64
+  RSA256_PUBLIC_KEY_BASE64: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/RSA256_PUBLIC_KEY_BASE64
+  RSA256_PRIVATE_KEY_BASE64: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/RSA256_PRIVATE_KEY_BASE64
```
Comment (gidsg) [inline copilot/fsd-frontend/manifest.yml:66]: Please remove this, only authenticator should need the private key.

Comment (aaronwilliamsv1) [reply copilot/fsd-frontend/manifest.yml]: Agreed. Removed

**Comment by aaronwilliamsv1** [comment]
> Frontend should not need private key, please remove this.

Removed

**Review by gidsg — APPROVED** [review]

**Pull request merged**